### PR TITLE
Add my card information

### PR DESCRIPTION
**Pull request by Jitesh-Gadage16**
Add a New card and modify the content with my info.

### CODE REVIEW

**Review by Preet-Sojitra — CHANGES_REQUESTED** [review]
You have added your card at the end of the file, you are supposed to add it to the top of the file beneath the comment saying `Paste your cards below this comment`. Move it upwards and then commit your changes and then push it, we will merge it after reviewing it.
If you need any help feel free to ask.
Thank you

**Review by nzaih1999 — CHANGES_REQUESTED** [review]
Hello  Jitesh...thank you for your contribution. Your card looks ready for merge but would you please move it to the top? It makes it much easier for us maintainers to work with the codebase

**Comment by CandidDeer** [comment]
Thank you so much for your contribution and making the requested changes! Well done! I have merged your PR now and your card will be live in a few moments!

**Pull request merged**
